arch/arm/core/aarch64/reset.S

```diff
@@ -93,19 +86,6 @@ SECTION_SUBSEC_FUNC(TEXT,_reset_section,__start)
 	* Zephyr entry happened in EL3. Do EL3 specific init before
 	* dropping to lower EL.
 	*/
-	/* Enable access control configuration from lower EL */
```

Comment:
Also for `ACTLR` I moved that in the platform-specific code because the register literally `Provides IMPLEMENTATIONDEFINED configuration and control options for EL3`. So it is definitely not something generic we want to have in the general reset code.